nebula-unity/Assets/scripts/DungeonMapGenerator.cs

```diff
@@ -120,6 +120,7 @@ public class PotentialDoor {
         map.transform.position = new Vector3(0f, 0f, 0f);
         Map mapScript = map.AddComponent<Map>();
         mapScript.SetMapSize(xSize, ySize);
+        mapSize = new Vector2Int(xSize, ySize);
```

Comment:
I just put this here to show you that all you need to do to fix your issue was set mapSize.  You can actually do this however you want.